vignettes/quickstart.Rmd

```diff
@@ -30,54 +30,33 @@ In particular the core functions are:
 
 ## The Data
 
-In order to demonstrate the package we will use a simulated dataset consisting of 120 
-patients with 3 visits per patient. The patients will be split into two treatment groups
-A and B. Group A will be our control group and group B our active group. The data is
-simulated such that the treatment effect has a linear slope of 4, i.e. the true treatment
-affect across our 3 visits is 0, 4 and 8. 
+In order to demonstrate the package we will use a publicly available example data set from an antidepressant clinical trial of an active drug versus placebo. The relevant endpoint is the Hamilton 17-item rating scale for depression (HAMD17) which was assessed at baseline and weeks 1, 2, 4, and 6. Study drug discontinuation occurred in 24% (20/84) for the active drug and 26% (23/88) for placebo. All data after study drug discontinuation are missing and there is a single additional intermittent missing observation.
 
-```{r setup}
+```{r}
 library(rbmi)
 library(dplyr)
-set.seed(6569)
-dat_full <- simulate_data(n = 120) %>% as_tibble()
-dat_full
-```
 
-We now introduce some missingness to our dataset. For the purpose of this vignette we will
-set visit 2 to missing completely at random with a 30% probability and visit 3 to missing
-completely at random with a 40% probability.  
-
-```{r}
-dat <- dat_full
-missing_index_vis2 <- rbinom(nrow(dat), 1, 0.3) == 1 & dat$visit == "visit_2"
-missing_index_vis3 <- rbinom(nrow(dat), 1, 0.4) == 1 & dat$visit == "visit_3"
-dat[missing_index_vis2, "outcome"] <- NA_real_
-dat[missing_index_vis3, "outcome"] <- NA_real_
-dat
+data("antidepressant_data")
+dat <- antidepressant_data
 ```
 
-Note that we have left the rows containing missing outcomes inside the dataset still. This
-is because rbmi expects its input dataset to be complete; that is that there must be 1 row
-per patient per visit even if the outcome value is missing. If your dataset is incomplete
-then the `expand_locf()` helper function can be used to add in any missing rows, using 
-LOCF imputation to impute the covariate values i.e:
+We consider an imputation model with the mean change from baseline in the HAMD17 score as the outcome (variable `change` in the dataset), included the treatment group (`THERAPY`), the (categorical) visit (`VISIT`), treatment-by-visit interactions, the baseline HAMD17 score (`basval`), and baseline HAMD17-by-visit interactions as covariates, and assumed a common unstructured covariance matrix in both groups. The chosen analysis model is ANCOVA which adjusts for the baseline HAMD17 value.
```

Comment:
Perhaps:

We consider an imputation model for mean change from baseline in the HAMD17 score (variable `change` in the dataset) dependent on: the treatment group (`THERAPY`); the (categorical) visit (`VISIT`); treatment-by-visit interactions; the baseline HAMD17 score ....

Reply:
MMM, maybe I like more as it is if it is not a strong preference for you